ngraph/test/type_prop_layers.cpp

```diff
@@ -74,31 +74,35 @@ TEST(type_prop_layers, interpolate)
                     .same_scheme(PartialShape{2, 2, Dimension::dynamic(), Dimension::dynamic()}));
 }
 
-TEST(type_prop_layers, interpolate_v3)
-{
-    using op::v3::Interpolate;
-    using InterpolateMode = op::v3::Interpolate::InterpolateMode;
-    using CoordinateTransformMode = op::v3::Interpolate::CoordinateTransformMode;
-    using InterpolateAttrs = op::v3::Interpolate::InterpolateAttrs;
-
-    auto image = make_shared<op::Parameter>(element::f32, Shape{2, 2, 33, 65});
-    auto dyn_output_shape = make_shared<op::Parameter>(element::i64, Shape{2});
-    auto output_shape = op::Constant::create<int64_t>(element::i64, Shape{2}, {15, 30});
-
-    InterpolateAttrs attrs;
-    attrs.axes = {2, 3};
-    attrs.mode = InterpolateMode::nearest;
-    attrs.coordinate_transformation_mode = CoordinateTransformMode::half_pixel;
-    attrs.antialias = false;
-    attrs.pads_begin = {0, 0, 0, 0};
-    attrs.pads_end = {0, 0, 0, 0};
-    auto op = make_shared<Interpolate>(image, output_shape, attrs);
-    ASSERT_EQ(op->get_shape(), (Shape{2, 2, 15, 30}));
-
-    EXPECT_TRUE(make_shared<Interpolate>(image, dyn_output_shape, attrs)
-                    ->get_output_partial_shape(0)
-                    .same_scheme(PartialShape{2, 2, Dimension::dynamic(), Dimension::dynamic()}));
-}
+// TEST(type_prop_layers, interpolate_v4)
```

Comment:
Please remove this test if you created new tests to check interpolate v4

Reply:
Done.